content/en/docs/setup/cri/cri-installation.md

```diff
@@ -0,0 +1,140 @@
+---
+reviewers:
+- vincepri
+- bart0sh
+title: CRI installation
+content_template: templates/concept
+weight: 100
+---
+{{% capture overview %}}
+Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
+This page contains installation instruction for various runtimes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+### Docker
+
+{{< tabs name="tab-cri-docker-installation" >}}
+{{< tab name="Ubuntu 16.04" codelang="bash" >}}
+# Install prerequisites.
+apt-get install apt-transport-https ca-certificates curl software-properties-common
+
+# Download GPG key.
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+
+# Add docker apt repository.
+add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
+
+# Install docker.
+apt-get update && apt-get install docker-ce=17.03.2~ce-0~ubuntu-xenial
```

Comment:
we are merging a PR to validate 18.06 by default:
https://github.com/kubernetes/kubernetes/pull/68495
[1]